apps/services/activities/src/app/worker/activities.processor.ts

```diff
@@ -0,0 +1,25 @@
+import { Process, Processor } from '@nestjs/bull'
+import { Job } from 'bull'
+import { Inject } from '@nestjs/common'
+import { ConfigType } from '@nestjs/config'
+import {
+  ActivitiesConfig,
+  activitiesQueueName,
+  sessionJobName,
+} from '../activities.config'
+import { SessionsService } from '../sessions/sessions.service'
+import { Session } from '../sessions/session.model'
+
+@Processor(activitiesQueueName)
+export class ActivitiesProcessor {
+  constructor(
+    private readonly sessionsService: SessionsService,
+    @Inject(ActivitiesConfig.KEY)
+    private config: ConfigType<typeof ActivitiesConfig>,
```

Comment:
Am I missing the usage or isn't the config unused?

Reply:
Correct, not used, removed.